0098-mobile-subdao-quality-of-service-requirements.md

```diff
@@ -99,6 +99,15 @@ See the table below for examples of how the multipliers will affect modeled cove
 | E    | 1,000     | 0.00X              |  1.00X               |  1.00X           | 0.00     |
 
 
+### Radio Health Metrics
+In order for CBRS Radios to earn rewards for the epoch, they must show the following for each category listed below:
+
+STATUS = Online
+SAS STATUS = Authorized
+CORE STATUS = Connected
+ON AIR = On
+
+If any of these health metrics does not equal the required output identified above for 12 or more heartbeats during an epoch, that radio will not earn PoC rewards for that epoch. 
```

Comment:
I'm a little concerned about this being very prescriptive in the HIP which will make it somewhat hard to update later.